src/components/Map/Layers/Buildings/HelsinkiBuildings/Popup.tsx

```diff
@@ -2,6 +2,51 @@ import React from 'react'
 import Feature from 'ol/Feature'
 import { Table, TableBody, TableCell, TableRow } from '@mui/material'
 import _ from 'lodash'
+// Variables
+var heatings
```

Comment:
Could you change all the "var" declarations in the code to "const" if they don't change? And to "let" if they are redeclared.